Gek62/hometask-4/my_array_rotate.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Array class extension created
+class Array
+  def lab_rotate(displacement = 1)
+    displacement = displacement % size
+    # rubocop:disable Style/SlicingWithRange
+    self[displacement..-1] + self[0...displacement]
+    # rubocop:disable Style/SlicingWithRange
```

Comment:
First of all, you probably wanted to reenable the previously disabled rule here, but instead, you're trying to disable it one more time here. That's why Hound is barking here.

But instead of disabling the rule, please update the code according to the Hound's previous complaints.

Reply:
One more thing, please use more descriptive commit messages. Now you have 3 commits with the same message and the message itself is a bit generic which is not good